README.md

```diff
@@ -131,29 +103,27 @@ Example usage:
 
 The SCARPE_TEST_CONTROL environment variable can contain a path to a test-control-interface script for the Webview display service. If you look at test_helper, it gives some examples of how to use it.
 
-## Definition of Done
-
-Scarpe is not intended to be a perfect replica of every element of Shoes. It is, however, intended to be functionally Shoes-compliant on mordern tooling. Certain benchmarks for this include:
-
-* [Parity with the Nobody Knows Shoes manual](https://github.com/whymirror/why-archive/raw/master/shoes/nobody-knows-shoes.pdf)
-  * This is our top priority
-* Most of our measurement will come from manually running the `examples/examples/examples_that_do_not_yet_work` directory, and moving them out into working as we pass them.
-* Able to run various external existing Shoes apps.
-  * [Shoes' native splash app](https://github.com/shoes/shoes-deprecated/blob/develop/lib/shoes.rb#L124-L176)
-  * [Example directory](https://github.com/shoes/shoes-deprecated/tree/develop/samples)
-  * [Found shoes apps](https://gist.github.com/search?l=Ruby&q=shoes.rb)
-* The ultimate test would be to run, functionally, [HacketyHack](https://github.com/whymirror/hacketyhack)
-
 ## More info
 
-* Great cheatsheet on pages 48-50 of [Nobody Knows Shoes](https://github.com/whymirror/why-archive/raw/master/shoes/nobody-knows-shoes.pdf)
+* [Nobody Knows Shoes manual](https://github.com/whymirror/why-archive/raw/master/shoes/nobody-knows-shoes.pdf)
 * [Original shoes (archived)](https://github.com/shoes/shoes-deprecated)
   - For now we are aiming towards _original shoes_, but I believe later can learn from Shoes3.
   - [wiki](https://github.com/shoes/shoes-deprecated/wiki)
 * [shoes3](https://github.com/shoes/shoes3)
   - [wiki](https://github.com/shoes/shoes3/wiki)
   * [Blog covering shoes3 history](https://web.archive.org/web/20190731215758/https://walkabout.mvmanila.com/)
 * [shoes4 (JRuby, incomplete)](https://github.com/shoes/shoes4)
+* [Shoes' native splash app](https://github.com/shoes/shoes-deprecated/blob/develop/lib/shoes.rb#L124-L176)
+* [Example directory](https://github.com/shoes/shoes-deprecated/tree/develop/samples)
```

Comment:
Maybe call this something like "original Shoes example directory" or "deprecated Shoes example directory" or something? Just "examples directory" sounds a lot like it's our examples directory, which is different.